deluge/core/torrentmanager.py

```diff
@@ -693,7 +693,7 @@ def _save_state(self):
                 os.fsync(_file.fileno())
         except (OSError, cPickle.PicklingError) as ex:
             log.error("Unable to save %s: %s", filename, ex)
-            return
+            return False
```

Comment:
I think without a concrete idea of return value, it's best left without so it's relatively obvious